.doc_gen/readmes/services.yml

```diff
@@ -0,0 +1,181 @@
+actions:
```

Comment:
It feels like this whole file could be gleaned from the metadata, rather than duplicating it here.
Otherwise, this is yet another metadata file that has to be maintained.